src/types/Identification.ts

```diff
@@ -0,0 +1,11 @@
+import { Enumeration } from "./index";
+
```

Comment:
Identifications can be for many different entities, Product, Party, Order etc. 

Should we rename this to PartyIdentification ? 

@dixitdeepak @adityasharma7

Reply:
Makes sense Sir

Reply:
Sir, Deleted this schema as renamed it to PartyIdentification schema and declared it in the Party.ts file instead of having a separate schema file.